core/src/test/java/org/apache/calcite/test/enumerable/EnumerableCorrelateTest.java

```diff
@@ -142,10 +142,10 @@ class EnumerableCorrelateTest {
     tester(false, new HrSchema())
         .query(
             "select empid, name from emps e where exists (select 1 from depts d where d.deptno=e.deptno)")
-        .explainContains(""
-            + "EnumerableCalc(expr#0..5=[{inputs}], empid=[$t0], name=[$t2])\n"
-            + "  EnumerableCorrelate(correlation=[$cor0], joinType=[inner], requiredColumns=[{1}])\n"
-            + "    EnumerableTableScan(table=[[s, emps]])\n"
+        .explainContains("EnumerableCalc(expr#0..3=[{inputs}], empid=[$t0], name=[$t2])\n "
```

Comment:
was the previous plan incorrect?
If not, can you provide a test case for which the previous plan was incorrect, but it is now correct?

Reply:
There is no change in the plan, it was just an indentation fix. I have reverted it in the latest commit just to avoid any confusion. Thanks

Reply:
The second half of my question stands.
If the RelFieldTrimmer was wrong, is there a test case where you can show the difference?

Reply:
Sorry, I assumed that you have looked at the new test case added in JdbcTest (testMultiLevelDecorrelation). If there is no RelFieldTrimmer fix then this new testcase fails with same exception as reported in the JIRA.

Reply:
Great, thank you, I was only looking at the last commit.